scripts/gitlab/test-linux.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# ARGUMENT $1 Rust flavor to run test with (stable/beta/nightly)
+# ARGUMENT $1 Rust flavor to run test with (stable/nightly)
```

Comment:
This script is for testing against different versions of *rust* isn't it, not parity-ethereum?